apps/web/next.config.js

```diff
@@ -322,29 +327,33 @@ const nextConfig = {
       // These rewrites are other than booking pages rewrites and so that they aren't redirected to org pages ensure that they happen in beforeFiles
       ...(isOrganizationsEnabled
         ? [
+            orgDomainMatcherConfig.root
```

Comment:
Allows to conditionally enable rewrite to organization for root path.

We disable it for single domain setup of org